FirebasePerformance/Sources/Loggers/FPRGDTEvent.h

```diff
@@ -15,24 +15,23 @@
 #import <Foundation/Foundation.h>
 
 #import <GoogleDataTransport/GoogleDataTransport.h>
+#import "FirebasePerformance/Sources/Protogen/nanopb/perf_metric.nanopb.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class FPRMSGPerfMetric;
-
 /**
- * Google Data Transport event wrapper used for converting Fireperf Proto object to
+ * Google Data Transport event wrapper used for converting Fireperf nanopb object to
  * GDTEvent object.
  */
 @interface FPRGDTEvent : NSObject <GDTCOREventDataObject>
 
-/** Perf metric that is going to be converted. */
-@property(nonatomic, readonly) FPRMSGPerfMetric *metric;
+/** firebase_perf_v1_PerfMetric that is going to be converted. */
+@property(nonatomic, readonly) firebase_perf_v1_PerfMetric metric;
 
 - (instancetype)init NS_UNAVAILABLE;
 
-/** Converts a PerfMetric event to a GDTEvent. */
-+ (instancetype)gdtEventForPerfMetric:(FPRMSGPerfMetric *)perfMetric;
+/** Converts a firebase_perf_v1_PerfMetric object to a GDTEvent. */
++ (instancetype)gdtEventForPerfMetric:(firebase_perf_v1_PerfMetric)perfMetric;
```

Comment:
Curious, NIT: Is there a way to replace `firebase_perf_v1_PerfMetric` to something more relatable. Eg. FPRPerfMetric? The name is too long to perceive :)

Reply:
This may result in a huge change throughout the SDK. I would prefer to make this change in a later PR.